src/engine/input/RunOnKeyAttribute.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+public class RunOnKeyAttribute : RunOnInputAttribute
```

Comment:
Add documentation and examples.
The reason for examples is that the input system will be widely used by new code and it will help to have documentation on it.